sycl/test-e2e/bindless_images/vulkan_interop/sampled_images.cpp

```diff
@@ -184,16 +184,17 @@ bool run_test() {
     cbbi.sType = VK_STRUCTURE_TYPE_COMMAND_BUFFER_BEGIN_INFO;
     cbbi.flags = VK_COMMAND_BUFFER_USAGE_ONE_TIME_SUBMIT_BIT;
 
-    VK_CHECK_CALL(vkBeginCommandBuffer(vk_computeCmdBuffer, &cbbi));
-    vkCmdPipelineBarrier(vk_computeCmdBuffer, VK_PIPELINE_STAGE_TOP_OF_PIPE_BIT,
+    VK_CHECK_CALL(vkBeginCommandBuffer(vk_compute_cmd_buffer, &cbbi));
+    vkCmdPipelineBarrier(vk_compute_cmd_buffer,
+                         VK_PIPELINE_STAGE_TOP_OF_PIPE_BIT,
                          VK_PIPELINE_STAGE_TRANSFER_BIT, 0, 0, nullptr, 0,
                          nullptr, 1, &barrierInput);
-    VK_CHECK_CALL(vkEndCommandBuffer(vk_computeCmdBuffer));
+    VK_CHECK_CALL(vkEndCommandBuffer(vk_compute_cmd_buffer));
```

Comment:
Merely changing code style isn't "improve vulkan interop ... testing" and should be done in a separate NFC PR.

Reply:
reverted to include only functional changes. other changes will be part of a separate PR as you suggested.